lib/facade_converter.ts

```diff
@@ -85,6 +85,31 @@ export class FacadeConverter extends base.TranspilerBase {
     return handler && !handler(pa);
   }
 
+  visitTypeName(typeName: ts.EntityName) {
+    if (typeName.kind !== ts.SyntaxKind.Identifier) {
+      this.visit(typeName);
```

Comment:
I don't actually have any idea, this is carried over from the prior implementation